api/src/test/java/com/messagebird/MessageBirdClientTest.java

```diff
@@ -642,11 +633,56 @@ public void testViewWebhook() throws UnauthorizedException, GeneralException, No
 
         when(messageBirdServiceMock.requestByID(VOICE_CALLS_BASE_URL + WEBHOOKS, "ANY_ID", WebhookResponseData.class))
                 .thenReturn(webhookResponseData);
-        final WebhookResponseData response = messageBirdClientInjectMock.viewWebHook("ANY_ID");
+        final WebhookResponseData response = messageBirdClientInjectMock.viewWebhook("ANY_ID");
         verify(messageBirdServiceMock, times(1)).requestByID(VOICE_CALLS_BASE_URL + WEBHOOKS,
                 "ANY_ID", WebhookResponseData.class);
         assertNotNull(response);
         assertEquals(response.getData().get(0).getId(), webhookResponseData.getData().get(0).getId());
         assertEquals(response.getData().get(0).getUrl(), webhookResponseData.getData().get(0).getUrl());
     }
+
+    @Test
```

Comment:
missing tests with negative scenarios

Reply:
Do you have any specific scenarios in mind? I don’t think we need to go crazy with tests considering that’s a simple REST wrapper and we testing against mocks anyway. Adding integration tests on the other hand would be beneficial imho.

Reply:
Yes, I agree integration test is beneficial, we should write them. On the other hand, in my opinion, unit tests should be written with TDD. So every scenario including negative ones needs to be covered. That's why I asked for negative scenarios. In here we are not writing tests before writing code so okay. if you think it is unnecessary, no need to write them.